src/core/recv_buffer.c

```diff
@@ -186,16 +267,21 @@ QuicRecvBufferResize(
     _In_ uint32_t TargetBufferLength
     )
 {
+    CXPLAT_DBG_ASSERTMSG(
+        RecvBuffer->RecvMode != QUIC_RECV_BUF_MODE_APP_OWNED,
+        "Should never resize in App-owned mode");
     CXPLAT_DBG_ASSERT(
         TargetBufferLength != 0 &&
         (TargetBufferLength & (TargetBufferLength - 1)) == 0); // Power of 2
-    CXPLAT_DBG_ASSERT(!CxPlatListIsEmpty(&RecvBuffer->Chunks)); // Should always have at least one chunk
+    // Should always have at least one chunk
```

Comment:
style nit: either keep this on the same line like it was (my preference) or you need to use the format:
```c
//
// Should always have at least one chunk
//
```